java/src/main/java/com/google/cloud/dataproc/templates/pubsublite/PubSubLiteToBigTable.java

```diff
@@ -0,0 +1,183 @@
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package com.google.cloud.dataproc.templates.pubsublite;
+
+import static com.google.cloud.dataproc.templates.util.TemplateConstants.*;
+
+import com.google.cloud.bigtable.data.v2.BigtableDataClient;
+import com.google.cloud.bigtable.data.v2.models.RowMutation;
+import com.google.cloud.dataproc.templates.BaseTemplate;
+import com.google.cloud.spark.bigquery.repackaged.org.json.JSONArray;
+import com.google.cloud.spark.bigquery.repackaged.org.json.JSONObject;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.TimeoutException;
+import org.apache.commons.lang3.StringUtils;
+import org.apache.spark.api.java.function.ForeachPartitionFunction;
+import org.apache.spark.api.java.function.VoidFunction2;
+import org.apache.spark.sql.Dataset;
+import org.apache.spark.sql.Row;
+import org.apache.spark.sql.SparkSession;
+import org.apache.spark.sql.streaming.StreamingQuery;
+import org.apache.spark.sql.streaming.StreamingQueryException;
+import org.apache.spark.sql.streaming.Trigger;
+import org.apache.spark.sql.types.DataTypes;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class PubSubLiteToBigTable implements BaseTemplate, java.io.Serializable {
+  private static final Logger LOGGER = LoggerFactory.getLogger(PubSubLiteToBigTable.class);
+  private String inputProjectID;
+  private String pubsubInputSubscription;
+  private String pubsubCheckpointLocation;
+  private long timeoutMs;
+  private int streamingDuration;
+  private String pubSubBigTableOutputInstanceId;
+  private String pubSubBigTableOutputProjectId;
+  private String pubSubBigTableOutputTable;
+  private final String sparkLogLevel;
+
+  public PubSubLiteToBigTable() {
+    inputProjectID = getProperties().getProperty(PUBSUB_INPUT_PROJECT_ID_PROP);
+    pubsubInputSubscription = getProperties().getProperty(PUBSUB_INPUT_SUBSCRIPTION_PROP);
+    timeoutMs = Long.parseLong(getProperties().getProperty(PUBSUB_TIMEOUT_MS_PROP));
+    streamingDuration =
+        Integer.parseInt(getProperties().getProperty(PUBSUB_STREAMING_DURATION_SECONDS_PROP));
+    pubsubCheckpointLocation = getProperties().getProperty(PUBSUBLITE_CHECKPOINT_LOCATION_PROP);
+    pubSubBigTableOutputInstanceId =
+        getProperties().getProperty(PUBSUB_BIGTABLE_OUTPUT_INSTANCE_ID_PROP);
+    pubSubBigTableOutputProjectId =
+        getProperties().getProperty(PUBSUB_BIGTABLE_OUTPUT_PROJECT_ID_PROP);
+    pubSubBigTableOutputTable = getProperties().getProperty(PUBSUB_BIGTABLE_OUTPUT_TABLE_PROP);
```

Comment:
Any specific reason, why are we reusing PUBSUB_* template properties?

Reply:
All the properties are applicable to both Pubsub and Pubsublite template except for it has one checkpoint variable in addition. With the intent for reducing redundant variables, i have reused. Moreover the names are also almost same except for pubsub and pubsublite. If there is any other name, i would have definitely used new variables. But if the standard is to create new variables, i can go ahead and create them. Pls let me know

Reply:
I believe, it is misleading.  e.g. pubsubInputSubscription (PUBSUB_INPUT_SUBSCRIPTION_PROP), it should be clear that it is not related to pubsub but pubsub lite.

Reply:
I would suggest renaming to convey the correct usage.

Reply:
Made the requested changes and commited